### ISSUE TITLE
Alert in Off-Canvas is not being read in Voice Over

### ISSUE THREAD

**Issue by ravenokko** [shopware/platform#17142]
### Shopware Version

trunk

### Affected area / extension

Platform(Default)

### Actual behaviour

When updating the cart (deleting products, adding promotions, etc.) the notification is not being read. This happens in success, warning/info and error states and is a violation of [WCAG SC 1.3.1](https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships.html)

<img width=30% height=30% alt="Image" src="https://github.com/user-attachments/assets/a94d6f6c-7e26-43cc-b3bb-1e9e944534eb" />

<img width=30% height=30% alt="Image" src="https://github.com/user-attachments/assets/9730ef65-9470-4762-b437-5aaae54f5e51" />

<img width=30% height=30% alt="Image" src="https://github.com/user-attachments/assets/00b01207-0c91-46ad-b2f7-ac7611684342" />

### Expected behaviour

Alert is read

### How to reproduce

There are several easy ways to reproduce this:

1. Add 2 different products to the cart and delete 1
2. Create a Product Bundle with a discount, add it to the cart, then remove 1 product
3. Try to add a promotion that doesn't exist

**Comment by tobiasberge** [comment]
Hi @ravenokko,

I can also reproduce this. Looks like this is a priority issue because the `.cart-live-update` container is always read due to "assertive", but the alerts only have "polite". And the `.cart-live-update` always overrules the alerts. I can take a look.

<img width="1424" height="1290" alt="Image" src="https://github.com/user-attachments/assets/c087fc78-64dd-4661-9c01-c38b70bd19c5" />
